File: Class_Lib/game_data/ClassMove.cs
using System;

namespace game_data
{
    public class Move
    {
        public string name;
        public int BP;
        public int PP;
        public int currentPP; 
        public string type;
        public string category;
        public bool contact;
        public bool spread;

        public Move(string name, int BP, int PP, string type, string category, bool contact, bool spread)
        {
            this.name = name;
            this.BP = BP;
            this.PP = PP;
            this.currentPP = PP; 
            this.type = type;
            this.category = category;
            this.contact = contact;
            this.spread = spread;
        }

        public void UseMove()
        {
            if (currentPP > 0)
                currentPP--;
        }
    }
}

File: PyLibs/ClassMove.py
class Move:
    def __init__(self, name, BP, PP, type, category, contact, spread):
        self.name = name
        self.BP = BP
        self.PP = PP
        self.currentPP = PP
        self.type = type
        self.category = category
        self.contact = contact
        self.spread = spread

    def use_move(self):
        if self.currentPP > 0:
            self.currentPP -= 1
